ospackage/recipes/buildcommand.go

```diff
@@ -46,10 +48,64 @@ func BuildCommand(step *osconfigpb.SoftwareRecipe_Step, artifacts map[string]str
 
 // StepFileCopy builds the command for a FileCopy step
 func StepFileCopy(step *osconfigpb.SoftwareRecipe_Step_FileCopy, artifacts map[string]string) ([]string, error) {
-	fmt.Println("StepFileCopy")
+	dest, err := normPath(step.FileCopy.Destination)
+	if err != nil {
+		return nil, err
+	}
+
+	permissions, err := parsePermissions(step.FileCopy.Permissions)
+	if err != nil {
+		return nil, err
+	}
+
+	if _, err := os.Stat(dest); err != nil {
+		if !os.IsNotExist(err) {
```

Comment:
minor: double negatives are hard to parse, can this be changed to `if os.IsExist(err)`?

Reply:
os.IsNotExist is oddly not the opposite of os.IsExist actually (despite the names seeming that way). Each of those only match a very specific set of errors (I think this is the relevant source: https://golang.org/src/os/error_unix.go)